### ISSUE TITLE
unclear how to use 'hdnes' skin

### ISSUE THREAD

**Issue by rob-lindman** [SourMesen/Mesen#204]
this may be a documentation / website issue. 

the app says it supports the hdnes skins, but it's not clear how to load them or to associate them with a given game. any nes games I have are in zip format, it would be cool if I could just drop the skin on the zip in explorer and if Mesen would detect this in the zip.

this is probably explained somewhere, but perhaps I am too tired to read it, I just want to play the exciting enhanced castlevania I saw on youtube which led me to this emulator, and i have the png file. 

also it would be exceptional if there was a simple way to export a skin file from an existing game... the mechanics would elude me, but perhaps dumping the character rom so that the existing images were saved as a png and then I could load up photoshop and mutate any game, then load that skin. perhaps this is already covered, but it was not obvious from the Mesen website, and I didn't see an issue here.

Thanks.

**Comment by GregoryMcGregerson** [comment]
Hi there. HD packs isn't as simple as having a single PNG file from a different HD pack method. You need txt files created with the Emulator HDNes as well for it to fiction. I'm not aware of the Castlevania HD pack being ported to HDNes but I could be wrong.

Anyways the way it works is explained in the video settings when you hover over the blue circle question mark. I will repost it here. First make sure "Use HDNes HD packs" is enabled by checking the tick box.

Explanation is as follows:
> HD Packs should be placed in the "HdPacks" folder in a subfolder matching the name of the ROM. e.g: MyRom.nes should have their HD Pack in "HdPacks\MyRom\hires.txt".

HDNes is a bit limited at the moment. I'm one of the only few people that experiment with that feature. If you want a certain game ripped in the HDNes format let me know I will do it for you. Then you can edit the graphics all you want in a single or multiple graphic sheets.

**Comment by SourMesen** [comment]
Like Gregory said above, that Castlevania HD pack isn't available as a HDNes pack - it's built as a hack in a custom version of Nestopia.  Though, now that I think about it, I might be able to make a tool to convert it to HDNes format relatively easily - I'll take a look at that.

Discussions about HD packs and the like have been on-going in this thread over at RomHacking.net: http://www.romhacking.net/forum/index.php?topic=24125.0

Like I mentioned in my posts throughout that thread, I'm currently in the middle of improving the HD pack feature in Mesen (by adding a way to dump graphic data, fixing existing bugs, adding new features to the HDNes format).  The work for this is pretty much done and part of the code on GitHub - just needs to be tested a bit more before I can make a new release of Mesen containing the updates.

**Comment by rob-lindman** [comment]
thanks for the info. I stumbled on this recent innovation on Youtube and was excited to try it out. 

I think whatever can be done to streamline the development, distribution, and launch / play scenario for this sort of thing will make it more pervasive. 

making folders and busy work like that will probably not be appealing to the masses, so whatever can be done to streamline it, such as putting the HDNES parts in a zip and a UI for selecting an HDNES pack and saving that option for loading / gameplay would certainly move it forward. 

I did obtain the hack of Nestopia, but I like the Mesen emulator more, and anything that supported this sort of functionality natively would be best of breed in my book.
